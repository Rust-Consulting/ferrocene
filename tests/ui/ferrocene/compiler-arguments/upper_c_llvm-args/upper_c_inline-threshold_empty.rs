//@ check-fail
//@ compile-flags: -C llvm-args=--inline-threshold

fn main() {}

// ferrocene-annotations: um_rustc_C_llvm_args
