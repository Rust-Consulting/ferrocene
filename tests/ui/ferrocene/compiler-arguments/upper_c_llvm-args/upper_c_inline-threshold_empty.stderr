rustc -Cllvm-args="..." with: for the --inline-threshold option: requires a value!
